src/main/java/cloud/fogbow/ms/core/service/WhiteList.java

```diff
@@ -1,24 +1,24 @@
 package cloud.fogbow.ms.core.service;
 
-import cloud.fogbow.common.util.HomeDir;
-import cloud.fogbow.ms.MembershipService;
-
-import cloud.fogbow.ms.constants.ConfigurationPropertyKeys;
-import org.apache.log4j.Logger;
-import cloud.fogbow.ms.constants.Messages;
-import cloud.fogbow.ms.constants.SystemConstants;
-
 import java.io.FileInputStream;
 import java.io.FileNotFoundException;
-import java.io.IOException;
 import java.io.InputStream;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Properties;
 
+import org.apache.log4j.Logger;
+
+import cloud.fogbow.common.util.HomeDir;
+import cloud.fogbow.ms.MembershipService;
+import cloud.fogbow.ms.constants.ConfigurationPropertyKeys;
+import cloud.fogbow.ms.constants.Messages;
+import cloud.fogbow.ms.constants.SystemConstants;
+
 public class WhiteList implements MembershipService {
```

Comment:
Shoudn't these exceptions be thrown above instead of making long try/catch/finally statements?